Dockerfile.template.erb

```diff
@@ -19,17 +19,31 @@ FROM alpine:3.9
 <% end %>
 <% else %>
 <% if is_armhf %>
+# To set multiarch build for Docker hub automated build.
+FROM golang:alpine AS builder
+WORKDIR /go
+RUN apk add curl --no-cache
+RUN curl -L https://github.com/balena-io/qemu/releases/download/v3.0.0%2Bresin/qemu-3.0.0+resin-arm.tar.gz | tar zxvf - -C . && mv qemu-3.0.0+resin-arm/qemu-arm-static .
```

Comment:
I think It's better to verify sha256sum of this tar file.

Reply:
Verify sha256sum in https://github.com/fluent/fluentd-docker-image/pull/203/commits/8f8bbb3fe64f9cdffc5feca3282e6955b951b2a2.